console/console-init/ui/src/modules/iot-device/components/DeviceList/DeviceList.tsx

```diff
@@ -61,7 +61,7 @@ export const DeviceList: React.FunctionComponent<IDeviceListProps> = ({
 }) => {
   const tableColumns: (string | ICell)[] = [];
   selectedColumns.forEach(column => {
-    switch (column) {
+    switch (column?.toLowerCase()) {
       case "deviceId":
```

Comment:
you will have to change deviceId in lower case as well otherwise this case will not work.
case "deviceid":